adoc/Release-Notes.adoc

```diff
@@ -31,6 +31,124 @@ endif::[]
 [id='sec.major-change']
 == Major Changes
 
+[id='sec.1_3_1']
+=== Release 1.3.1, February 2019
+
+[id='sec.1_3_1.new']
+==== What Is New?
+
+* {scf} has been updated to version 2.15.2:
+** Default PodSecurityPolicies (PSPs) come with the helm charts
+** cflinuxfs3 now available as a stack
+** Added nginx buildpack
+** Support added for placement zones & isolation segments
+* The Stratos UI has been updated to version 2.3:
+** Support for extensions
+** For a full list of features and fixes see https://github.com/SUSE/stratos/releases/tag/2.3.0.
+
+For information about deploying and administering {product}, see the product manuals at
+{doc-url}.
+
+
+[id='sec.1_3_1.feature']
+==== Features and Fixes
+
+* App-autoscaler no longer dependent on hairpin
+* Credhub on Azure is now supported
+* Corrected service name to work with `syslog` drains
+* Certificates rely on correct UAA FQDN
+* Removed obsolete key and diego-cell readiness probe from role-manifest.yml
+* Changed one variable name to align with upstream practices--this may require changes to sizing:
+** `cf-routing` replaces `routing`
+* Includes these {cf} component versions:
+** app-autoscaler: 1.0.0
+** bpm: 1.0.0
+** capi: 1.66.0
+** cf-deployment: 3.6.0
+** cf-mysql: 36.15.0
+** cf-routing: 0.180.0
+** cf-sle12: 1.52.6
+** cf-smoke-tests: 40.0.6
+** cf-syslog-drain: 7.0
+** cf-usb: 1.0.1
+** cflinuxfs2: 1.266.0
+** cflinuxfs3: 0.60.0
+** credhub: 2.0.2
+** diego: 2.16.0
+** garden-runc: 1.16.3
+** groot-btrfs: 1.0.4
+** kubectl: 1.9.6
+** loggregator: 103.1
+** loggregator-agent: 2.0
+** nats: 25
+** nfs-volume: 1.2.0
+** opensuse42: 1.8.6
+** postgres-release: 26
+** scf-helper: 1.0.1
+** cf-acceptance-tests: 2.8
+** statsd-injector: 1.3.0
+** uaa: 60.2
+** uaa-fissile: c9edf895
+* Buildpacks:
+** binary-buildpack: 1.0.30
+** dotnet-core-buildpack: 2.0.3
+** go-buildpack: 1.8.33
+** java-buildpack: 4.17.2
+** nginx-buildpack: 1.0.8
+** nodejs-buildpack: 1.6.43
+** php-buildpack: 4.3.70
+** python-buildpack: 1.6.27
+** ruby-buildpack: 1.7.31
+** staticfile-buildpack: 1.4.39
+
+
+[id='sec.1_3_1.issue']
+==== Known Issues
+
+* For deployments on Amazon EKS: the The {awsa} Service Broker(https://aws.amazon.com/partners/servicebroker/) should now be used instead of the deprecated `cf-brokers` wrapper.
+
+* For custom PSPs, `SYS_RESOURCE` no longer needs to specified under added capabilities in the role-manifest.yml
```

Comment:
It should be the scf-config-values.yaml instead of role-manifest.yml?

Reply:
Yup, wrong file.